docs/source/tensorbay_cli/cli_commands.rst

```diff
@@ -1,3 +1,159 @@
 ##############
  CLI Commands
 ##############
+
+The following table lists the currently supported CLI commands.(:numref:`Table. %s <cli_commands_table>`).
+
+.. _cli_commands_table:
+
+.. table:: CLI Commands
+   :align: center
+   :widths: auto
+
+   =======================================================  ===========================================================
+   Commands                                                 Description
+   =======================================================  ===========================================================
+   :ref:`tensorbay_cli/cli_commands:gas dataset`            **gas dataset** deals with TensorBay datasets.
+   :ref:`tensorbay_cli/cli_commands:gas ls`                 **gas ls** deals with list operation.
+   :ref:`tensorbay_cli/cli_commands:gas cp`                 **gas cp** deals with copy operation.
+   :ref:`tensorbay_cli/cli_commands:gas rm`                 **gas rm** deals with remove operation.
+   :ref:`tensorbay_cli/cli_commands:gas draft`              **gas draft** deals with draft.
+   :ref:`tensorbay_cli/cli_commands:gas commit`             **gas commit** deals with commit operation.
+   =======================================================  ===========================================================
+
+*************
+ gas dataset
+*************
+
+**gas dataset** deals with TensorBay datasets.
+
+Create a dataset.
+
+.. code:: console
+
+   gas dataset [dataset_name]
+
+Note that there is no tb: before the dataset name when creating a dataset.
+
+List all datasets.
+
+.. code:: console
+
+   gas dataset
+
+Delete a dataset.
+
+.. code:: console
+
+   gas dataset -d tb:[dataset_name]
+
+
+*********
+ gas ls
+*********
+
+**gas ls** deals with list operation.
+
+List the segments of a dataset.(default revision)
+
+.. code:: console
+
+    gas ls tb:[dataset_name]
+
+List the segments of a specific dataset revision.
+
+.. code:: console
+
+   gas ls tb:[dataset_name]@[revision]
+
+List the segments of a specific dataset draft.
+
+.. code:: console
+
+    gas ls tb:[dataset_name]#[draft_number]
+
+List all data of a segment.
+
+.. code:: console
+
+    gas ls tb:[dataset_name]:[segment_name]
+    gas ls tb:[dataset_name]@[revision]:[segment_name]
+    gas ls tb:[dataset_name]#[draft_number]:[segment_name
+
+Get certain data.
+
+.. code:: console
+
+    gas ls tb:[dataset_name]:[segment_name]://[remote_path]
+    gas ls tb:[dataset_name]@[revision]:[segment_name]://[remote_path]
+    gas ls tb:[dataset_name]#[draft_number]:[segment_name]://[remote_path]
+
+
+********
+ gas cp
+********
+**gas cp** deals with copy operation.
+
+Upload the all files to a segment.
```

Comment:
Upload file(s) to a segment. The ``local path`` refers to a file or a directory.